analyses/focal-cn-file-preparation/05-define-most-focal-cn-units.Rmd

```diff
@@ -155,7 +155,7 @@ filtered_consensus_cytoband_status <- consensus_seg_cytoband_status %>%
   ) %>%
   # Filter the annotated CN data to include only neutral chromosome arms and disagreements
   filter(
-    dominant_arm_status %in% c("neutral", "uncallable", "unstable") |
+    dominant_arm_status %in% c("neutral", "unstable") &
```

Comment:
The filtering step here was taking the all `neutral`, `unstable`, and `uncallable` arms and providing the cytoband status for these arms. Instead, we wanted to take provide only the `neutral` and `unstable` arms to this step so that we can get the cytoband status for these arms and leave the `uncallable` arms.

That being said, for each uncallable region, all samples are now labeled `uncallable`.

Reply:
As I originally understood the logic here, it was possible to have an `uncallable` arm that contained callable cytoband (i.e. an arm that was 70% uncallable could still have many callable bands), so we wanted to include those.

Reply:
Ah okay, that's reasonable. I'll revert the change to the logic. Re the comment above:
> Without in-depth knowledge of the steps to obtain the uncallable status calls, I might expect that uncallable counts = total number of samples. That is not always the case - is this expected based on upstream steps?

I believe the reason the total number of samples for an uncallable region may not add up is because all samples may not have data for that region? (I am trying to track back and look at the raw data and how many samples that have data for each region)

Were the inconsistent regions the X and/or Y chromosomes? The `Xp` arm is where I noticed the inconsistency described.

Reply:
> Were the inconsistent regions the X and/or Y chromosomes? The Xp arm is where I noticed the inconsistency described.

The short answer is I don't remember - looks like you just reverted this change, is there anything you would consider an inconsistency?

Reply:
So thinking about the logic a little more and @jashapiro's comment,

For sample X, cytoband 11q11 may be `uncallable` because more than 50% of the region is uncallable for that sample.

For sample Y on the other hand, cytoband 11q11 may be `loss` because more than 90% of the region for that sample is loss.

This can only be done for samples that have copy number data for that specific region (hence some regions may not have data for all 915 samples). I used the cytoband data for an example, but the same goes for the arm level calls, although arm level calls would be expected to have more agreement between samples due to the region being broader? (this is just a guess).

So based on that logic, I do not believe that the uncallable counts would necessarily be equal to the total number of samples, does that make sense?

That being said, it appears that the `Xp` and `Xq` arms are the only chromosome arms to have less samples than the total number of samples that have been labeled `uncallable`, but they are no other competing calls for these arms (by this I mean, `uncallable` is the only unique label in the `Xp` and `Xq` regions).